### PR TITLE
fix(other): kubernetes error by degrading node version from v20.7.0 to v20.2.0

### LINKED ISSUE #6776: 💥 [DevOps] Webapp - Fix the Dockerfile Labels
## 💥 DevOps ticket

Fix the Dockerfile Labels

Here in the webapp dockerfile we have still the backend in it.

```dockerfile
# Labels
LABEL org.label-schema.build-date="${BUILD_DATE}"
LABEL org.label-schema.name="ocelot.social:backend"
LABEL org.label-schema.description="Web Frontend of the Social Network Software ocelot.social"
LABEL org.label-schema.usage="https://github.com/Ocelot-Social-Community/Ocelot-Social/blob/master/README.md"
LABEL org.label-schema.url="https://ocelot.social"
LABEL org.label-schema.vcs-url="https://github.com/Ocelot-Social-Community/Ocelot-Social/tree/master/backend"
LABEL org.label-schema.vcs-ref="${BUILD_COMMIT}"
LABEL org.label-schema.vendor="ocelot.social Community"
LABEL org.label-schema.version="${BUILD_VERSION}"
LABEL org.label-schema.schema-version="1.0"
LABEL maintainer="devops@ocelot.social"
```

### LINKED ISSUE #6778: 💥 [DevOps] Fix Kubernetes Error by Degrading Node Version to v20.2.0
## 💥 DevOps ticket

Fix Kubernetes Error by Degrading Node Version to v20.2.0

### Additional Information

´´´bash
yarn run v1.22.19
$ cross-env NODE_OPTIONS=--openssl-legacy-provider nuxt start
env: can't execute 'node': Text file busy
error Command failed with exit code 126.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
´´´

### PR DESCRIPTION
**Pull request by Tirokk**
## 🍰 Pullrequest

Kubernetes error by degrading node version from v20.7.0 to v20.2.0

### Issues

- fixes #6778
- fixes #6776 

### Todo

- [X] None

### CODE REVIEW

File: neo4j/Dockerfile
Comment (Tirokk) [inline neo4j/Dockerfile:1]: Is changing this labels an issue @ulfgebhardt ?

File: webapp/Dockerfile [hunk 1]
```diff
@@ -22,11 +22,11 @@ ENV PORT="3000"
...
-LABEL org.label-schema.name="ocelot.social:backend"
+LABEL org.label-schema.name="ocelot.social:webapp"
 LABEL org.label-schema.description="Web Frontend of the Social Network Software ocelot.social"
 LABEL org.label-schema.usage="https://github.com/Ocelot-Social-Community/Ocelot-Social/blob/master/README.md"
 LABEL org.label-schema.url="https://ocelot.social"
-LABEL org.label-schema.vcs-url="https://github.com/Ocelot-Social-Community/Ocelot-Social/tree/master/backend"
+LABEL org.label-schema.vcs-url="https://github.com/Ocelot-Social-Community/Ocelot-Social/tree/master/webapp"
 LABEL org.label-schema.vcs-ref="${BUILD_COMMIT}"
 LABEL org.label-schema.vendor="ocelot.social Community"
 LABEL org.label-schema.version="${BUILD_VERSION}"
```
Comment (Tirokk) [inline webapp/Dockerfile:32]: Is changing this labels an issue @ulfgebhardt ?

File: webapp/Dockerfile.maintenance [hunk 2]
```diff
@@ -22,11 +22,11 @@ ENV PORT="3000"
...
-LABEL org.label-schema.name="ocelot.social:backend"
+LABEL org.label-schema.name="ocelot.social:maintenance"
 LABEL org.label-schema.description="Maintenance page of the Social Network Software ocelot.social"
 LABEL org.label-schema.usage="https://github.com/Ocelot-Social-Community/Ocelot-Social/blob/master/README.md"
 LABEL org.label-schema.url="https://ocelot.social"
-LABEL org.label-schema.vcs-url="https://github.com/Ocelot-Social-Community/Ocelot-Social/tree/master/backend"
+LABEL org.label-schema.vcs-url="https://github.com/Ocelot-Social-Community/Ocelot-Social/tree/master/webapp"
 LABEL org.label-schema.vcs-ref="${BUILD_COMMIT}"
 LABEL org.label-schema.vendor="ocelot.social Community"
 LABEL org.label-schema.version="${BUILD_VERSION}"
```
Comment (Tirokk) [inline webapp/Dockerfile.maintenance:32]: Is changing this labels an issue @ulfgebhardt ?

File: backend/Dockerfile [hunk 3]
```diff
@@ -1,7 +1,7 @@
 ##################################################################################
 # BASE (Is pushed to DockerHub for rebranding) ###################################
 ##################################################################################
-FROM node:20.7.0-alpine3.17 as base
+FROM node:20.2.0-alpine3.17 as base
```
Comment (mahula) [inline backend/Dockerfile:4]: This change is not necessary.
The [last update here](https://github.com/Ocelot-Social-Community/Ocelot-Social/pull/6748) was already from `20.6.0-alpine3.17` to `20.7.0-alpine3.17`.
The Node issue we have in Kubernetes seems to be the container, which contains this repo.

File: webapp/Dockerfile [hunk 4]
```diff
@@ -1,7 +1,7 @@
 ##################################################################################
 # BASE (Is pushed to DockerHub for rebranding) ###################################
 ##################################################################################
-FROM node:20.7.0-alpine3.17 as base
+FROM node:20.2.0-alpine3.17 as base
```
Comment (mahula) [inline webapp/Dockerfile:4]: This change is not necessary.
The [last update here](https://github.com/Ocelot-Social-Community/Ocelot-Social/pull/6749) was already from `20.6.0-alpine3.17` to `20.7.0-alpine3.17`.
The Node issue we have in Kubernetes seems to be the container, which contains this repo.

**Review by mahula — CHANGES_REQUESTED** [review]

File: webapp/Dockerfile (hunk 4 above)
Comment (Tirokk) [reply webapp/Dockerfile:4]: I don't get it.
What we have to change then?

The error is on Kubernetes in the webapp and this is used there. Nothing else as far I know.

File: backend/Dockerfile (hunk 3 above)
Comment (Tirokk) [reply backend/Dockerfile:4]: The container is based on this as far as I see.

File: webapp/Dockerfile (hunk 4 above)
Comment (mahula) [reply webapp/Dockerfile:4]: My bad. 
Ignore the comment. Your changes are correct.

File: backend/Dockerfile (hunk 3 above)
Comment (mahula) [reply backend/Dockerfile:4]: My bad. 
Ignore the comment. Your changes are correct.

**Review by mahula — APPROVED** [review]

**Review by Mogge — APPROVED** [review]

**Review by mahula — APPROVED** [review]

**Pull request merged**